deployment/ansible/roles/nih-wayfinding.otp-data/defaults/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+## Defined in azavea.opentripplanner
+#otp_bin_dir: /opt/opentripplanner
+#otp_data_dir: /var/otp
+#otp_jarfile: "./target/otp-0.13.0.jar"
+
```

Comment:
I suppose the intent is to take this commented stuff out once the role is modified?